src/app/components/home/home.component.ts

```diff
@@ -18,14 +20,37 @@ import { StoresService } from 'src/app/services/stores.service';
 })
 export class HomeComponent {
   readonly categories$: Observable<ProductCategoryModel[]> =
-    this._productCategoriesService.getAllCategories().pipe(shareReplay(1));
+    this._productCategoriesService.getAllCategories();
 
-  readonly stores$: Observable<StoreModel[]> = this._storesService
-    .getAllStores()
-    .pipe(shareReplay(1));
+  readonly stores$: Observable<StoreModel[]> =
+    this._storesService.getAllStores();
+
+  readonly fruitsAndVegetablesList$: Observable<ProductModel[]> =
+    this._productsService.getAllProducts().pipe(
+      shareReplay(1),
```

Comment:
shareReplay should be on the end in pipe

Reply:
ok, I will update.